frontend/e2e/pages/metrics/config-step.ts

```diff
@@ -2,6 +2,7 @@ import { config as configStepData } from '../../fixtures/create-new/config-step'
 import { METRICS_STEP_SAVING_FILENAME } from '../../fixtures';
 import { downloadFileAndCheck } from '../../utils/download';
 import { expect, Locator, Page } from '@playwright/test';
+import { format } from 'e2e/utils/date-time';
```

Comment:
`e2e` is defined in the tsconfig?
if not, PLZ use relative path

Reply:
ok